nixos/doc/manual/development/replace-modules.xml

```diff
@@ -0,0 +1,48 @@
+<section xmlns="http://docbook.org/ns/docbook"
+        xmlns:xlink="http://www.w3.org/1999/xlink"
+        xmlns:xi="http://www.w3.org/2001/XInclude"
+        version="5.0"
+        xml:id="sec-replace-modules">
+
+<title>Replace Modules</title>
+
+<para>It's also possible to disable modules, removing the option
```

Comment:
nit: Avoid starting new paragraph with `It`, maybe use the passive form.  I suggest reading the current documentation and try to mimic the current style to write these paragraph the same way.